src/Tribe/Venue.php

```diff
@@ -698,7 +698,9 @@ public static function get_fetch_callback( $event ) {
 		}
 
 		return static function () use ( $event ) {
-			$venue_ids = array_map( 'absint', (array) get_post_meta( $event, '_EventVenueID' ) );
+			$venue_ids = array_filter(
+				array_map( 'absint', (array) get_post_meta( $event, '_EventVenueID' ) )
+			);
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 1b778b7e5ae8741ae57f690bb9d8c8f5</sub>